_posts/2019-08-02-max-quant.md

```diff
@@ -0,0 +1,18 @@
+---
+site: freiburg
+title: MaxQuant goes Galaxy
+tags: [tools]
+---
+
+
+After lots of requests for a more up-to-date MaxQuant tool it finally is there! The European Galaxy server now contains two MaxQuant tools.
+[One](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/maxquant/maxquant/1.6.3.4) features options for SILAC and label free
+quantification and also the possibility to process
+fractioned data by uploading an experimental design template.
+[The other one](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/maxquant_mqpar/maxquant_mqpar/1.6.3.4) operates on a parameter
+file uploaded by the user. This enables the usage of all the features of MaxQuant, but might be less convenient to use.
+
+Those tools have been developed by Damian Glaetzer and Friedel Drepper from the University of Freiburg and are available in the Galaxy ToolShed for everyone.
+Thanks a lot Damian and Friedel!
+
+The work was presented at the [GCC2019](https://galaxyproject.org/events/gcc2019/) and the poster is [here]().
```

Comment:
@dglaetzer is that PR ok for you? Do you have a PDF of your poster?

Reply:
Looks good :)
The poster isn't published, but the slides are:
https://f1000research.com/slides/8-1162
I forgot to select the Galaxy gateway, but i hope they still can change it...
If you think the poster is better, i can send it to you via email. GitHub doesn't allow me to upload it...

Reply:
Just found the poster on the conference website:
https://static.sched.com/hosted_files/gcc2019/75/Poster_15C_GCC2019_MaxQuant_Galaxy_20190701_print.pdf

Reply:
Cool, danke. I have added those to the PR.

Let me know if this is ok for you.

Reply:
I'm ok with it.